### PR TITLE
Update to wgpu 0.5.0

### PR DESCRIPTION
**Pull request by rytone**

### CODE REVIEW

**Comment by Yatekii** [comment]
Thanks for the work!
Do you also want to bump the version of this crate? So we can make a release?
It's a real shame we once f-ed up and bumped one too many so now we are out of sync ...

File: src/lib.rs [hunk 1]
```diff
@@ -43,10 +36,7 @@ impl Shaders {
     }
 
     fn get_program_code() -> (&'static str, &'static str) {
-        (
-            include_str!("imgui.vert"),
-            include_str!("imgui.frag"),
-        )
+        (include_str!("imgui.vert"), include_str!("imgui.frag"))
```
Comment (Yatekii) [inline src/lib.rs:39]: Why did this get changed?

Comment (rytone) [reply src/lib.rs:39]: Not exactly sure, I have my editor configured to run `rustfmt` on save, and a lot of stuff seemed to be reformatted as a result. Do you use custom settings for `rustfmt`? (I just use the defaults)

**Comment by Yatekii** [comment]
I am sorry, I will close this in favor of #12. I hope you understand. Thanks for your work =)

**Comment by rytone** [comment]
Totally understandable. To be honest, I was thinking about making some of the exact changes that are in the other PR (particularly removing the dependency on `glsl-to-spirv`), but I decided to just pump this out first; fortunately someone came along with a bit more motivation than me. :laughing:

**Pull request closed without merging**